sdk/src/builtins.rs

```diff
@@ -1,4 +1,4 @@
-//! Solana builtin helper macros
+//! Solana helper macros for declaring builtin programs.
```

Comment:
```suggestion
//! Solana helper macros for declaring built-in programs.
```
I don't think we're consistent with this, but this is my preference